pylint/checkers/classes/class_checker.py

```diff
@@ -525,6 +525,12 @@ def _has_same_layout_slots(
         "beginning with an underscore) is access outside the class or a "
         "descendant of the class where it's defined.",
     ),
+    "W0213": (
+        "Flag member <%s.%s: %d> overlaps with another",
```

Comment:
```suggestion
        "Flag member <%s.%s: %d> implicitely hide <%s.%s: %d> as %s|%s=%s.",
```

Reply:
```suggestion
        "Flag member <%s.%s: %d> implicitly hides <%s.%s: %d> as %s|%s=%s.",
```

(This is my new specialty :D)

Reply:
Adapted these suggested changes slightly - I found it easier to understand the cause when shown the intersection (`A & B`) rather than the union (`A | B`) of bitmasks where this issue can occur.